jquery/Script/StudentManagerJquery.js

```diff
@@ -0,0 +1,378 @@
+var studentArray = [];
+var baseUrl = 'https://localhost:44330/api/';
+
+//#region Generic Ajax  crud operation On student Table 
+
+function StudentListGeneric() {
+    debugger;
+    // var stdList =GetStudentListApi;//JSON.parse(localStorage.getItem("student"));
+    // studentArray=
+    serverCall(baseUrl+"Student", "get", null, successg);
+
+    // ajax start to get all student and put records in Table
+    function successg(result) {
+        var appendRow = "";
+        result.forEach(function (value, index) {
+
+            appendRow += "<tr><td>" + value.student.Name + "</td>" + "<td>" + value.student.FName + "</td>" + "<td>" + value.student.Email + "</td>" + "<td>" + value.student.Phone + "</td>" + "<td>" + value.student.Dob + "</td>" + "<td>" + value.student.Password + "</td>"+"<td>"+value.CoursesCount+"</td>";
+            appendRow += "<td><button class='editBtn btn btn-primary'  data-id=" + value.Id + ">Edit</button>";
+            appendRow += "<button style='margin-left:5px;' class='editNewWindowBtn btn btn-secondary'  data-id=" + value.Id + ">Edit With Courses</button>";
+            appendRow += "<button style='margin-left:5px;' class='deleteBtn btn btn-danger'  data-id=" + value.Id + ">Delete</a></td></tr>";
+            //tBody.innerHTML += appendRow;
+
+        });
+        $('#stdList').html('').append(appendRow);
+        $('.editNewWindowBtn').click(function () {
+         var id=$(this).attr('data-id');
+         localStorage.setItem('StudentId',id);
+         window.location.href='EditStudent.html';
+        });
+        $('.deleteBtn').click(function () {
+            debugger;
+            var id = parseInt($(this).attr('data-id'));
+            var url = baseUrl + 'Student?id=' + id;
```

Comment:
move baseUrl into server methods and concatenate there. No need to send from each method and its obvious